diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/files/NewlinesRuleWarnTest.kt

```diff
@@ -26,7 +26,7 @@ class NewlinesRuleWarnTest : LintTestBase(::NewlinesRule) {
             mapOf("maxCallsInOneLine" to "1"))
     )
     private val ruleId = "$DIKTAT_RULE_SET_ID:${NewlinesRule.NAME_ID}"
-    private val dotQuaOrSafeAccessOrPostfixExpression = "${WRONG_NEWLINES.warnText()} should be split before second and other dot/safe access"
+    private val dotQuaOrSafeAccessOrPostfixExpression = "${WRONG_NEWLINES.warnText()} should be split before second and other dot/safe access after first call expression"
```

Comment:
I think we shouldn't add this to warning text. This part makes text too complex, and also we don't (or at least shouldn't) forbid splitting these expressions. The main reason in our logic with splitting only after call expression is that we can't distinguish fully-qualified names from chain of property access expressions. I.e., this is still valid:
```
project
    .projectDir
    .absolutePath
```

Reply:
done